server/http4s-server/src/main/scala/sttp/tapir/server/http4s/http4s.scala

```diff
@@ -1,3 +1,15 @@
 package sttp.tapir.server
 
-package object http4s extends TapirHttp4sServer
+import sttp.capabilities.fs2.Fs2Streams
+import sttp.model.sse.ServerSentEvent
+import sttp.tapir.{CodecFormat, Schema, StreamBodyIO, streamBody}
+
+import java.nio.charset.Charset
+
+package object http4s extends TapirHttp4sServer {
+  def serverSentEventBody[F[_]]: StreamBodyIO[fs2.Stream[F, Byte], fs2.Stream[F, ServerSentEvent], Fs2Streams[F]] = {
```

Comment:
SSE stands for `server sent events` so I'd keep plural here as well

Reply:
done :)